pkg/templates/openapi/default.go

```diff
@@ -42,41 +44,80 @@ func GenSchemaDefaultPatch(ctx context.Context, schema *openapi3.Schema) ([]byte
 					group: name,
 				}
 			case len(nodes) == 1:
-				np := nodes[0].GetPatch()
-				if len(np) == 0 {
+				def := nodes[0].GetDefault()
+				if def == nil {
 					resultChan <- patchResult{
 						group: name,
 					}
 
 					return
 				}
 
+				// Generate patch.
+				var b []byte
+
+				p, err := sjson.SetBytes(b, nodes[0].GetID(), def)
+				if err != nil {
+					resultChan <- patchResult{
+						group: name,
+						err:   fmt.Errorf("set patch node error for %s %s: %w", nodes[0].GetID(), string(p), err),
+					}
+				}
+
```

Comment:
can we construct a path bytes without using sjson? if no, here also need to return inside the `err != nil` block.